src/components/shared/binaryCheckRadio.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
```

Comment:
please keep the name convention for the file `binary-check-radio`

Reply:
done